resources/module_annex_change_history.adoc

```diff
@@ -0,0 +1,126 @@
+[[change_history]]
+[appendix,obligation=informative]
+== Change history
+
+[[general]]
+=== General
+
+This annex documents the history of technical modifications made to ISO/TS 10303-42.
```

Comment:
@manuelfuenmayor this document identifier is probably not for all documents?

Reply:
@ronaldtse well, it seems like `<<general>>` is not present in any document.
But perhaps we can change that anchor to `[[change_history_general]]`?

Reply:
Yes please!